src/DetailsView/Styles/detailsview.scss

```diff
@@ -228,6 +231,70 @@ div.insights-file-issue-details-dialog-container {
         margin-top: 30px;
         font-size: 21px;
     }
+    .footer {
+        margin-bottom: 16px;
+        margin-top: 8px;
+        padding-top: 16px;
+        padding-right: 16px;
+        border-top: #edebe9 1px solid;
```

Comment:
This `border-top` color can be used from the common colors as well. Please check `colors.scss` to find a color that has a `high-contrast` color to go with that.